shims/common/src/main/scala/org/apache/gluten/config/GlutenConfig.scala

```diff
@@ -1241,6 +1243,16 @@ object GlutenConfig {
       .booleanConf
       .createWithDefault(false)
 
+  val COLUMNAR_MEMORY_UNTRACKED =
+    buildConf("spark.gluten.memory.untracked")
+      .internal()
+      .doc(
+        "When enabled, turn all native memory allocations in Gluten into untracked. Spark " +
```

Comment:
based on the description, this feature should be effect only with `DYNAMIC_OFFHEAP_SIZING_ENABLED` case? or do you intend to introduce this feature in case with static off-heap also?

Reply:
It doesn't relate to the off-heap sizing feature, the idea use case is to allow user set 

```
spark.memory.offHeap.enabled=false
spark.gluten.memory.untracked=true
```

to bypass allocation tracking from Spark memory manager.